compiled/tesoro/manifests/tesoro-rbac.yml

```diff
@@ -33,11 +33,9 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: tesoro
 subjects:
   - kind: ServiceAccount
-    name: tesoro
-    namespace: tesoro
+    namespace: default
```

Comment:
is `namespace: default` correct or shouldn't it be still the components name `tesoro`?